app/web/components/CookieBanner.tsx

```diff
@@ -71,6 +71,7 @@ export default function CookieBanner() {
         <IconButton
           aria-label={t("close")}
           onClick={(e) => handleClose(e, "button")}
+          size="large"
```

Comment:
Mui changed the default size so you'll see these added some places. This was done by their codemod script that automates as much of the migration as was possible.